### PR TITLE
add contribute section

### PR DESCRIPTION
**Pull request by mikeshng**
Signed-off-by: Mike Ng <ming@redhat.com>

@swopeb one of the feedback we got from the playback is that there is no "Contribute" section on our website. Turns out it's sort of hidden in the "Community" part. I created a dedicated section for "Contribute" with more contents and copied some Community stuffs over. But I feel like there is still some overlapping with "Community". Could you please take a look and comment? Thanks.

### CODE REVIEW

**Comment by mikeshng** [comment]
image

**Comment by mikeshng** [comment]
and this is our existing Community page https://open-cluster-management.io/community/resource

I moved `Get Involved` and `Certificate of Origin` to the new "Contribute" side

**Comment by mikeshng** [comment]
Most of the new contents I took from https://www.okd.io/#contribute

File: content/contribute/contribute.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+weight: -20
+---
+
+We know you've got great ideas for improving Open Cluster Management. So roll up your sleeves and come join us in the community!
+
+## Get Started
+
+All contributions are welcome! Open Cluster Management uses the Apache 2 license and does not require any contributor agreement to submit patches. Please open issues for any bugs or problems you encounter, ask questions in the #open-cluster-mgmt on Kubernetes Slack Channel, or get involved joining the open-cluster-management google mailing group.
+
+[Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0)
```
Comment (qiujian16) [inline content/contribute/contribute.md:12]: why we put a license bullet here, you could reference the link above.

File: content/contribute/contribute.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: Contribute to Open Cluster Management 
+weight: -20
+---
+
+We know you've got great ideas for improving Open Cluster Management. So roll up your sleeves and come join us in the community!
+
+## Get Started
+
+All contributions are welcome! Open Cluster Management uses the Apache 2 license and does not require any contributor agreement to submit patches. Please open issues for any bugs or problems you encounter, ask questions in the #open-cluster-mgmt on Kubernetes Slack Channel, or get involved joining the open-cluster-management google mailing group.
```
Comment (qiujian16) [inline content/contribute/contribute.md:10]: We do require developer to realize DCO and signoff

File: content/contribute/contribute.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- Join the [Mailing group](https://groups.google.com/g/open-cluster-management)
+- Attend an upcoming [Community meetings](https://github.com/open-cluster-management/community/projects/1)
+- Fork the [repository](https://github.com/open-cluster-management) you wish to work on
+- Get started by following the `CONTRIBUTING.md` guide in the repository you wish to work on
+
+## Get Involved / File a Bug
+
+Check out the project and consider contributing. You can pick up an issue to work on, or report a bug by creating a new issue [here](https://github.com/open-cluster-management/community/issues). When your code is ready to be reviewed, you can propose a pull request. You can find a good guide about the GitHub workflow [here](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project).
+
+If you intend to propose an enhancement or a new feature in open-cluster-management, submit a pull request in the [enhancements repo](https://github.com/open-cluster-management/enhancements). The proposal should follow the [kep format](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md).
```
Comment (qiujian16) [inline content/contribute/contribute.md:27]: let's change to use the kep format link here https://github.com/open-cluster-management/enhancements/blob/master/guidelines/README.md

File: content/contribute/contribute.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+[Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0)
+
+[Mailing group](https://groups.google.com/g/open-cluster-management)
+
+## Contribute to the Source
+
+- Join the [Mailing group](https://groups.google.com/g/open-cluster-management)
+- Attend an upcoming [Community meetings](https://github.com/open-cluster-management/community/projects/1)
+- Fork the [repository](https://github.com/open-cluster-management) you wish to work on
```
Comment (qiujian16) [inline content/contribute/contribute.md:20]: should we list some repos here? e.g. API/registration etc?

File: content/contribute/contribute.md (hunk 1 above)
Comment (mikeshng) [reply content/contribute/contribute.md]: done as suggested

File: content/contribute/contribute.md (hunk 2 above)
Comment (mikeshng) [reply content/contribute/contribute.md]: removed the part where it says does not require any agreement

File: content/contribute/contribute.md (hunk 3 above)
Comment (mikeshng) [reply content/contribute/contribute.md]: done as suggested

File: content/contribute/contribute.md (hunk 4 above)
Comment (mikeshng) [reply content/contribute/contribute.md]: done as suggested

**Pull request merged**